### PR TITLE
Flash the input field when inserting readable ID by click.

### PR DESCRIPTION
**Pull request by mmcdermott**

### CODE REVIEW

File: src/components/metrics/card/MetricCardViewSection/index.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import React, {Component} from 'react'
 
+import $ from 'jquery'
```
Comment (OAGr) [inline src/components/metrics/card/MetricCardViewSection/index.js]: probably not needed?

File: src/components/distributions/editor/style.css [hunk 2]
```diff
@@ -1,12 +1,18 @@
...
   @mixin small-input;
 }
 
 .Guesstimate .TextInput {
   line-height: 1.3em;
   font-size: 0.95em;
+  border-radius: 2px;
+  padding: 0;
+  border: 2px solid transparent;
+  transition: border-color $flash-duration-sec ease, background-color $flash-duration-sec ease;
```
Comment (OAGr) [inline src/components/distributions/editor/style.css]: you could also say ```transition: all```

File: src/components/distributions/editor/TextForm/TextInput.js [hunk 3]
```diff
@@ -222,8 +239,8 @@ export class TextInput extends Component{
   }
 
   render() {
-    const [{hasErrors, width, value, validInputs}, {editorState}] = [this.props, this.state]
-    const className = `TextInput ${width}` + (_.isEmpty(value) && hasErrors ? ' hasErrors' : '')
+    const {props: {hasErrors, width, value, validInputs}, state: {isFlashing, editorState}} = this
```
Comment (OAGr) [inline src/components/distributions/editor/TextForm/TextInput.js]: nice

File: src/components/metrics/card/MetricCardViewSection/index.js (hunk 1 above)
Comment (mmcdermott) [inline src/components/metrics/card/MetricCardViewSection/index.js]: good catch, will address momentarily.

**Pull request merged**